docs/modules/user-guide/partials/con_using-a-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using a top-level attribute
+
+[role="_abstract"]
+
+Use top-level attributes in devfiles to reuse global attributes in different sections of the same devfile.
```

Comment:
Let's be consistent with our verb choice and let's also name the value of reusing global attributes. Something like this: 

`Add a top-level attributes to your devfile to reuse global attributes in different sections of the same devfile. Reusing these attributes makes it easier to modify your devfile for your various development needs.` 

OR 

`Add a top-level attributes to your devfile to reuse global attributes in different sections of the same devfile. Reusing these attributes makes it easier to collaborate with different teammates.` 

Get a SME perspective to adequately capture the benefit of reusing global attributes.